.env

```diff
@@ -0,0 +1 @@
+SPRINGWOLF_VERSION=0.13.0
```

Comment:
This file could directly be `0.13.0-SNAPSHOT` and remove the need for the `-Dsnapshot=true` flag, but that requires some further changes in the release process.

The "standard" Maven release process is usually:
- Remove the `SNAPSHOT` suffix
- Release
- Increase version + add the `SNAPSHOT` suffix

I leave this change open for discussion

Reply:
On second thoughts, I see the benefit of it and like suggestion.
We will merge this, feel free to suggest a change, to further simplify this.

I enjoy all the thoughts and ideas you bring to this project.

Reply:
Based on this issue https://github.com/springwolf/springwolf-core/issues/295 it will reduce confusion which version is currently used.
So, one more argument for your suggestion :)